packit/service/web_hook.py

```diff
@@ -58,6 +58,30 @@ def github_release():
         logger.debug(f"/webhooks/github/release received ping event: {msg['hook']}")
         return "Pong!"
 
+    return _give_event_to_steve(msg)
+
+
+@app.route("/webhooks/github/pull_request")
+def github_pr():
+    msg = request.get_json()
+
+    if not msg:
+        logger.debug(
+            "/webhooks/github/pull_request: We haven't received any JSON data."
+        )
+        return "We haven't received any JSON data."
+
+    allowed_actions = ["opened", "edited", "reopened"]
```

Comment:
+ synchronised, that's when someone pushes new changes to the PR

I think that edited means when someone updates the original PR comment: not confusing at all, right?